.eslintignore

```diff
@@ -0,0 +1 @@
+*_data.js
```

Comment:
This seems ok for now, but it would make more sense to just delete this file. I think we can do that as a part of this PR? @lm-n

Reply:
@MathuraMG This file seems crucial for the interceptors, no? The global `allData` is used in all of the interceptors.

Reply:
oh no! I'm sorry, ignore this, I had it confused with [loadData](https://github.com/processing/p5.accessibility/issues/56). Apologies.

Reply:
I can delete that one, regarding `_data.js`; it just seemed to be a minified file that was really long. Do you think i should lint it?

Reply:
Let's delete that in a separate PR as originally planned. 
No, you're right, let's not lint `_data.js`. It's a file we actually copy over from the p5.js library, so it'll keep changing